tests/chaos/oadp/conftest.py

```diff
@@ -79,3 +80,80 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
     with node_mgmt_console(node=vm_node, node_mgmt="drain"):
         wait_for_node_schedulable_status(node=vm_node, status=False)
         yield vm_node
+
+
+@pytest.fixture()
+def rhel_vm_with_dv_running_factory(
```

Comment:
why is this fixture needed? why not use `tests.data_protection.oadp.conftest.rhel_vm_with_data_volume_template`?